_posts/2015-08-18-Proud-to-Announce-our-Series-A.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Announcing our $6.8M Series A"
+excerpt: "Double-down on the awesome."
+layout: post
+date: 2015-08-18 09:00:00 -0900
+author: jakub
+published: YES
+comments: true
+---
+
+We at Apiary are very excited (and a little bit proud) to announce that earlier this year we've closed our **Series A funding at $6.8M**. As an incredible show of trust for the company, all our existing seed investors doubled-down on their previous investment with David Aronoff from [Flybridge Capital][Flybridge] leading a strong new round to carry Apiary to our next stage of awesome.
+
+You might be wondering what we want to use the money for. In short, we plan on continuing what we do best&mdash;**lead the way for API tools** that help developers build APIs faster, easier and better. We've coincided the announcement today with an announcement about [adding support for testing][Dredd article] &amp; continuous integration&mdash;something we've been itching to do for some time now (as you might have previously observed in our Dredd blog posts). And I'm sure there are many more exciting additions on the way. As Apiary is gaining traction and becoming mainstream, we've also introduced features for our large customers&mdash;from support for Github for Enterprise, to the ability to check your API blueprints against a style guide. If there are 50+ devs where you work, you should [check this out][Enterprise].
+
+Since our last funding announcement Apiary has grown substantially&mdash;from 25,000 APIs to over 160,000 APIs; and from 7 employees in one room in Prague to over 30 people in Prague, San Francisco and other places around the world. Today, my thanks go out to [this team][Team], for their hard work and passion, and to you&mdash;our user community, all 130,000 of you, that have supported us and cheered on us every step of the way. Thank you! Děkujeme!
+
+Jakub Nešetřil, Apiary CEO
+
+[Flybridge]: http://www.flybridge.com
+[Dredd article]: http://blog.apiary.io/
+[Enterprise]: http://enterprise.apiary.io/
+[Team]: http://apiary.io/team
```

Comment:
Not sure if it matters, but this is a redirect to https://apiary.io/company#team